src/App.js

```diff
@@ -1,15 +1,28 @@
 import React from 'react';
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+} from 'react-router-dom';
 
 import './App.css';
 import Nav from './components/Nav';
-// import Home from './pages/Home';
-// import Maths from './pages/Maths';
+import Home from './pages/Home';
+import Maths from './pages/Maths';
 import Quote from './pages/Quote';
 
 const App = () => (
   <div className="App">
     <Nav />
-    <Quote />
+    <Router>
+      <Routes>
+        <Route path="/quote" element={<Quote />} />
+        <Route path="/maths" element={<Maths />} />
+        <Route path="/home" element={<Home />} />
```

Comment:
- [ ] (OPTIONAL) This path of `/home` is not necessary since the `Home` element already has the default path `/` Kindly remove that extra path 👍